src/test/java/nl/tudelft/ewi/devhub/webtests/ProjectPullTest.java

```diff
@@ -102,13 +106,18 @@ public void testCreatePullRequest() {
 			.get(0).click()
 			.listBranches()
 			.get(1);
-		
+
+
 		assertEquals(BRANCH_NAME, newBranch.getName());
-		
+
 		// navigate to pull request view
-		PullRequestView view = newBranch.click().openCreatePullRequestView();
-		assertThat(view.getCurrentUrl(), CoreMatchers.containsString("courses/ti1705/TI1705/groups/1/pull/1"));
-		
+		PullRequestView pullRequestView =
+			newBranch.click().openCreatePullRequestView();
+
+		// Grab the pull request instance for BRANCH_NAME from the DB.
+		PullRequest pullRequest = pullRequests.findOpenPullRequest(groupRepository, "refs/heads/" + BRANCH_NAME).get();
+
+		// TODO Assertions on pullRequestView against pullRequest
```

Comment:
I have pulled the `PullRequest` instance from the DB, so you can inspect that instance as well 😄